ci/requirements/minimum-tests.yml

```diff
@@ -1,6 +1,7 @@
 name: climpred-minimum-tests
 channels:
   - conda-forge
+  - numba/label/dev # Temporary fix to get Numba on Python 3.9
```

Comment:
You may want to merge this PR once latest version is available on `conda-forge`

Reply:
I‘d say we can merge this PR now. Or wait until numba is also available on main and delete that line